smithy-typescript-ssdk-libs/server-common/src/errors.ts

```diff
@@ -29,37 +41,57 @@ export const isFrameworkException = (error: any): error is SmithyFrameworkExcept
   return error.$frameworkError;
 };
 
-export class InternalFailureException implements SmithyException {
+export class InternalFailureException extends SmithyException {
   readonly name = "InternalFailure";
   readonly $fault = "server";
   readonly statusCode = 500;
```

Comment:
I think we decided we don't need this. Though could be done is separate PR if that's what you had planned.